proto/heimdallv2/clerk/query.proto

```diff
@@ -39,6 +39,13 @@ service Query {
   rpc IsClerkTxOld(RecordSequenceRequest) returns (IsClerkTxOldResponse) {
     option (google.api.http).get = "/clerk/is-old-tx";
   }
+
+  // GetLatestStateSyncId queries the latest state-sync id from L1.
+  rpc GetLatestStateSyncId(LatestStateSyncIdRequest)
+      returns (LatestStateSyncIdResponse) {
+    option (cosmos.query.v1.module_query_safe) = true;
```

Comment:
Please do not use `module_query_safe` here as it is actually not safe (it relies on 3rd party - the L1 contract)